dev-libs/highway/highway-1.2.0.recipe

```diff
@@ -52,9 +63,10 @@ INSTALL()
 	make -C build install
 
 	prepareInstalledDevelLibs \
-		libhwy libhwy_contrib
+		libhwy libhwy_contrib libhwy_test
```

Comment:
Nitpicking :) You could move these libraries into separate lines

Reply:
I know... Usually it's done if the line gets really long... But it also helps if the recipe gets constantly edited... However it doesn't allow appending to the list without changing at least one extra line.